web/src/ui_init.js

```diff
@@ -653,6 +652,7 @@ export function initialize_everything() {
     message_view_header.initialize();
     server_events.initialize();
     user_status.initialize(user_status_params);
+    compose_recipient.initialize();
```

Comment:
Why this is initialized twice? I am assuming it is by mistake, but just want to confirm. This is causing problems in implementing the keyboard behavior to open on focus.

Reply:
Oh yup a mistake, thanks for catching that!

Reply:
Fixed in most recent psuh